src/components/forms/form-elements/FormElements.vue

```diff
@@ -136,35 +136,56 @@
         <va-card :title="$t('forms.selects.title')">
```

Comment:
Probably makes more sense to move search here. So that we don't have to rescale select.

![image](https://user-images.githubusercontent.com/5394573/60444078-23366080-9c25-11e9-9cca-f32af6590934.png)